storage/entry_cache_test.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2014 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Kevin GuanJian (keynovo@gmail.com)
+
+package storage
+
+import (
+	"bytes"
+	"math"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/util/leaktest"
+	"github.com/coreos/etcd/raft/raftpb"
+)
+
+// TestRaftEntryCacheGet
```

Comment:
Test functions don't need comments unless you have something to say (`golint` doesn't care about comments in test files)